db/seeds.rb

```diff
@@ -14,25 +14,26 @@
 
 items.each do |row|
   row = row.to_h
+  row[:status] = row[:status].to_i
   Item.create!(row)
 end
 
-
-Tag.create(name: "Infused")
-Tag.create(name: "Black")
-Tag.create(name: "Red")
-Tag.create(name: "Pink")
-Tag.create(name: "Flake")
-Tag.create(name: "Coarse")
-Tag.create(name: "Fine")
-Tag.create(name: "accessory")
-
-
-
-salt  = Tag.create(name: "salt")
-
-item_collection = Item.all
-
-item_collection.each do |item|
-  item.tags << salt
-end
+ Tag.create(name: "Infused")
+ Tag.create(name: "Black")
+ Tag.create(name: "Red")
+ Tag.create(name: "Pink")
+ Tag.create(name: "Flake")
+ Tag.create(name: "Coarse")
+ Tag.create(name: "Fine")
+ Tag.create(name: "accessories")
+ Tag.create(name: "salt")
+ 20.times do
+  Tag.create(name: Faker::Address.unique.country)
+ end
+
```

Comment:
Sorry to jump in on the seed file, @dphilla and @mikeyduece , I changed it so we had more tags, and more than one tag per item.

Reply:
No worries. We just did it that way so they all had at least one, since we need to add the functionality for the admin to add tags anyway